src/app/shared/palette/base-field/field-write.component.ts

```diff
@@ -54,6 +55,12 @@ export class FieldWriteComponent extends AbstractFieldWriteComponent implements
     if (this.caseField.field_type.id === 'AddressGlobal') {
       component.instance['ignoreMandatory'] = true;
     }
+    if (this.isSearchField && (
```

Comment:
mmm I have some doubts on this. This breaks encapsulation, field-write.component.ts should be generic and know nothing about the existence of a Address type